File: output/csbc_semish_2026/deepseek_deepseek_v3_2/zero/python/Cobra coral.py

import sys

def main():
    data = sys.stdin.read().strip().split()
    if len(data) != 4:
        return
    a, b, c, d = map(int, data)
    
    # Padrão verdadeiro: ...BVBPBVBPBVBP... (repetição de 4 cores)
    # Para 4 números consecutivos: posições 1 e 4 iguais, posições 2 e 3 diferentes entre si e diferentes de 1/4
    # Padrão falso: ...BVPBVPBVPBVP... (repetição de 3 cores)
    # Para 4 números consecutivos: posições 1 e 4 diferentes
    
    if a == d and b != c and a != b and a != c:
        print("V")
    else:
        print("F")

if __name__ == "__main__":
    main()

File: output/csbc_semish_2026/deepseek_deepseek_v3_2/zero/cpp/Cobra coral.cpp

#include <iostream>
using namespace std;

int main() {
    int a, b, c, d;
    cin >> a >> b >> c >> d;
    
    // Padrão verdadeiro: ...BVBPBVBPBVBP... -> sequência de 4: B V B P
    // Isso significa que a == c e b != d
    // Padrão falso: ...BVPBVPBVPBVP... -> sequência de 4: B V P B
    // Isso significa que a == d e b != c
    
    if (a == c && b != d) {
        cout << "V" << endl;
    } else {
        cout << "F" << endl;
    }
    
    return 0;
}
